pandas/tests/resample/test_datetime_index.py

```diff
@@ -112,6 +112,12 @@ def test_resample_integerarray():
                       dtype="Int64")
     assert_series_equal(result, expected)
 
+    result = ts.resample('3T').mean()
+    expected = Series([1, 4, 7],
+                      index=pd.date_range('1/1/2000', periods=3, freq='3T'),
+                      dtype='Int64')
```

Comment:
@jreback @jorisvandenbossche why returning Int64 here?  I would expect float64 or Float64.

e.g. if we do `ts[-1] += 1` before the resample, the mean comes back as float64.

Reply:
Yes, this should be Float64, because it is only accidentally that the results are all integer-like. 
This is one of the cases that I listed in https://github.com/pandas-dev/pandas/issues/37494